packages/client/webapp/src/views/document.vue

```diff
@@ -0,0 +1,219 @@
+<template>
+	<div class="doc-view-container">
```

Comment:
We are using `<section>` for Views containers.

Reply:
done